pvlib/test/test_irradiance.py

```diff
@@ -177,9 +177,11 @@ def test_perez_components():
         index=times
     )
     df_components = pd.DataFrame(components)
```

Comment:
Your test function uses a series as an input, so it gets a DataFrame as an output. I think the perez function itself currently has the correct behavior.